ag_data/presets/helpers.py

```diff
@@ -0,0 +1,104 @@
+from ag_data import models
+from ag_data.presets import sample_user_data
+from ag_data import utilities
+
+
```

Comment:
Optional:
First of this file overall looks good to me. But most of the lines of this file is doing simple things. Checking boundary and creating a model.

In other words, it doesn't have to be 104 lines. 104 lines matter because not only reviewer but also potential maintainer have to read this whole file to make sure it doesn't do any surprising stuffs.

- Are this helpers for use of test cases? Then why don't you merge this with `test_presets_helpers.py`.
- Comments are good to have but not much valuable information. You know python has already `IndexError: list index out of range` and consider what values are added by wrapping those exception with our code.

Reply:
- This file is indeed doing the things mostly used for example data setup. So it is used for the simulator, and tests. I agree this file can be optimized. What about calling this file `preset_data_generators.py` and place it in the original folder still?
- For the comments, which are so long that they are of almost equal length of the actual code. I planned to delete them, but have not got a chance. What do you think?